go-controller/pkg/ovn/gateway_test.go

```diff
@@ -57,6 +57,9 @@ func generateGatewayInitExpectedNB(testData []libovsdb.TestData, expectedOVNClus
 		Name:     gwRouterPort,
 		MAC:      util.IPAddrToHWAddr(joinLRPIPs[0].IP).String(),
 		Networks: networks,
+		Options: map[string]string{
+			"gateway_mtu": fmt.Sprintf("%d", util.GetMaxFrameLength()),
```

Comment:
[A new feature was added](https://github.com/ovn-org/ovn/commit/acffcc0dde6c641aeaf516e0b600813c7bc4d489) to upstream OVN recently, `gateway_mtu_bypass`, we should probably use it to bypass check_pkt_larger checks for types of traffic that don't need it, for example for TCP.